contracts/paratii/TcrPlaceholder.sol

```diff
@@ -0,0 +1,169 @@
+pragma solidity ^0.4.18;
+
+import "./ParatiiToken.sol";
+
+/// WARNING , NOT PRODUCTION READY, based on ADCHAIN Registry.
```

Comment:
Perhaps add the link https://github.com/skmgoldin/tcr/blob/master/contracts/Registry.sol here?

Reply:
I was actually using this https://github.com/AdChain/AdChainRegistry/blob/master/contracts/Registry.sol
😄